pkgs/factor-lang/default.nix

```diff
@@ -50,11 +49,11 @@ stdenv.mkDerivation rec {
     # zcat ${factorimage} | (cd $out/lib && tar -xvpf - factor/factor.image )
     cp ${factorimage} $out/lib/factor/factor.image
 
-    cp -r basis core extra unmaintained $out/lib/factor
+    cp -r basis core extra $out/lib/factor
```

Comment:
Is unmaintained gone?

Reply:
Yes, it's gone into a separate repository.